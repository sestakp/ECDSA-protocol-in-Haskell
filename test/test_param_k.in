Curve {
    p: 0xfffffffffffffffffffffffffffffffffffffffffffffffffffffffefffffc2f
    a: 0
    b: 7
    g: Point {
    x: 0x79be667ef9dcbbac55a06295ce870b07029bfcdb2dce28d959f2815b16f81798
    y: 0x483ada7726a3c4655da4fbfc0e1108a8fd17b448a68554199c47d08ffb10d4b8
}
    n: 0xfffffffffffffffffffffffffffffffebaaedce6af48a03bbfd25e8cd0364141
    h: 1
}