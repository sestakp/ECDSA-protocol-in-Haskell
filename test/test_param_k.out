Key {
d: 0x828f61640d9d585b06d4a45fc986e331ea4e2c81e2af7f9af316c597895736c8
Q: 0x04ab97bfa6a2afa9777647fd394553a1a7b494f817b14529468de1b5247915535240b9acbf058b0389ab35689e98337643002f28f0ca402e88dc25a2c9d41de132
}